pkg/apis/core/v1beta1/suite_test.go

```diff
@@ -0,0 +1,65 @@
+package v1beta1
```

Comment:
What is the intention of this test?

Reply:
The intention of this test is to exercise the creation and validation == expected result.

While in the validation package, we test the validation function as single functions.